

<!doctype html>
<html lang="en" class="no-js">
  <head>
    

<meta charset="utf-8">
<!-- Favicon Links Begin -->
<link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicon/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
<link rel="manifest" href="/assets/images/favicon/site.webmanifest">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" content="#ffffff">
<!-- Favicon Links End -->



<!-- begin SEO -->









<title>Publications - Ying Maâ€™s Personal Website</title>







<meta property="og:locale" content="en-US">
<meta property="og:site_name" content="Ying Ma's Personal Website">
<meta property="og:title" content="Publications">


  <link rel="canonical" href="https://YingMa0107.github.io/publications/">
  <meta property="og:url" content="https://YingMa0107.github.io/publications/">







  

  












  <script type="application/ld+json">
    {
      "@context" : "http://schema.org",
      "@type" : "Person",
      "name" : "Ying Ma",
      "url" : "https://YingMa0107.github.io",
      "sameAs" : null
    }
  </script>






<!-- end SEO -->


<link href="https://YingMa0107.github.io/feed.xml" type="application/atom+xml" rel="alternate" title="Ying Ma's Personal Website Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="https://YingMa0107.github.io/assets/css/main.css">

<meta http-equiv="cleartype" content="on">
    

<!-- start custom head snippets -->

<link rel="apple-touch-icon" sizes="57x57" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="60x60" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="72x72" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="76x76" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="114x114" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="120x120" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="144x144" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="152x152" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="180x180" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
<link rel="manifest" href="/assets/images/favicon/site.webmanifest">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" content="#ffffff">
<link rel="stylesheet" href="https://YingMa0107.github.io/assets/css/academicons.css"/>
<script type="text/x-mathjax-config"> MathJax.Hub.Config({ TeX: { equationNumbers: { autoNumber: "all" } } }); </script>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/latest.js?config=TeX-MML-AM_CHTML' async></script>

<!-- end custom head snippets -->

    <!-- Favicon Links Begin -->
    <link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicon/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
    <link rel="manifest" href="/assets/images/favicon/site.webmanifest">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="theme-color" content="#ffffff">
  </head>

  <body>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->
    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <button><div class="navicon"></div></button>
        <ul class="visible-links">
          <li class="masthead__menu-item masthead__menu-item--lg"><a href="https://YingMa0107.github.io/">Ying Ma's Personal Website</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://YingMa0107.github.io/publications/">Publications</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://YingMa0107.github.io/talks/">Talks</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://YingMa0107.github.io/research/">Research</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://YingMa0107.github.io/software/">Software</a></li>
          
        </ul>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>

    <style>
  .card {
    box-shadow: 2px 2px 5px #888888 inset;
  }
  
  .card-img-top {
    max-height: 300px;
    object-fit: contain;
  }

  .card-body {
    margin: 20px;
  }

  .card-title {
    margin-bottom: 10px;
  }

  .card-text {
    margin-bottom: 5px;
    text-align: justify;
  }

  .card-link {
    margin-top: 10px;
  }
</style>

<div id="main" role="main">
  <div class="archive">
    
      <h1 class="page__title">Publications</h1>
    

    <div class="row row-cols-1 row-cols-xl-2">
      
        
      
        
      
        
          <div class="col mb-4">
            <div class="card h-100 d-flex flex-column justify-content-between bg-light">
              <div class="card-body">
                <h2 class="card-title">Spatially informed cell type deconvolution for spatial transcriptomics</h2>
                <img src="https://YingMa0107.github.io/images/CARD_NBT_2022.jpg" class="card-img-top" alt="Spatially informed cell type deconvolution for spatial transcriptomics">
                <p class="card-text">Many spatially resolved transcriptomic technologies do not have single-cell resolution but measure the average gene expression for each spot from a mixture of cells of potentially heterogeneous cell types. Here, we introduce a deconvolution method, conditional autoregressive-based deconvolution (CARD), that combines cell-type-specific expression information from single-cell RNA sequencing (scRNA-seq) with correlation in cell-type composition across tissue locations. Modeling spatial correlation allows us to borrow the cell-type composition information across locations, improving accuracy of deconvolution even with a mismatched scRNA-seq reference. CARD can also impute cell-type compositions and gene expression levels at unmeasured tissue locations to enable the construction of a refined spatial tissue map with a resolution arbitrarily higher than that measured in the original study and can perform deconvolution without an scRNA-seq reference. Applications to four datasets, including a pancreatic cancer dataset, identified multiple cell types and molecular markers with distinct spatial localization that define the progression, heterogeneity and compartmentalization of pancreatic cancer.</p>
                <p class="card-text"><em>Ying Ma and Xiang Zhou</em></p>
                <p class="card-text text-danger"><strong></strong></p>
                <p class="card-text"></p>
              </div>
              <div class="card-footer px-0 mx-auto text-center w-100">
                <p class="card-link text-nowrap"><strong><a href="https://www.nature.com/articles/s41587-022-01273-7">Nature Biotechnology 2022</a></strong></p>
              </div>
            </div>
          </div>
        
      
        
      
        
      
        
      
        
          <div class="col mb-4">
            <div class="card h-100 d-flex flex-column justify-content-between bg-light">
              <div class="card-body">
                <h2 class="card-title">Integrative differential expression and gene set enrichment analysis using summary statistics for scRNA-seq studies</h2>
                <img src="https://YingMa0107.github.io/images/iDEA_NC_2020.png" class="card-img-top" alt="Integrative differential expression and gene set enrichment analysis using summary statistics for scRNA-seq studies">
                <p class="card-text">Differential expression (DE) analysis and gene set enrichment (GSE) analysis are commonly applied in single cell RNA sequencing (scRNA-seq) studies. Here, we develop an integrative and scalable computational method, iDEA, to perform joint DE and GSE analysis through a hierarchical Bayesian framework. By integrating DE and GSE analyses, iDEA can improve the power and consistency of DE analysis and the accuracy of GSE analysis. Importantly, iDEA uses only DE summary statistics as input, enabling effective data modeling through complementing and pairing with various existing DE methods. We illustrate the benefits of iDEA with extensive simulations. We also apply iDEA to analyze three scRNA-seq data sets, where iDEA achieves up to five-fold power gain over existing GSE methods and up to 64% power gain over existing DE methods</p>
                <p class="card-text"><em>Ying Ma, Shiquan Sun, Xuequn Shang, Evan T Keller, Mengjie Chen, Xiang Zhou</em></p>
                <p class="card-text text-danger"><strong></strong></p>
                <p class="card-text"></p>
              </div>
              <div class="card-footer px-0 mx-auto text-center w-100">
                <p class="card-link text-nowrap"><strong><a href="https://www.nature.com/articles/s41467-020-15298-6">Nature communications 2020</a></strong></p>
              </div>
            </div>
          </div>
        
      
        
          <div class="col mb-4">
            <div class="card h-100 d-flex flex-column justify-content-between bg-light">
              <div class="card-body">
                <h2 class="card-title">Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis</h2>
                <img src="https://YingMa0107.github.io/images/DR_GB_2019.png" class="card-img-top" alt="Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis">
                <p class="card-text">Dimensionality reduction is an indispensable analytic component for many areas of single-cell RNA sequencing (scRNA-seq) data analysis. Proper dimensionality reduction can allow for effective noise removal and facilitate many downstream analyses that include cell clustering and lineage reconstruction. Unfortunately, despite the critical importance of dimensionality reduction in scRNA-seq analysis and the vast number of dimensionality reduction methods developed for scRNA-seq studies, few comprehensive comparison studies have been performed to evaluate the effectiveness of different dimensionality reduction methods in scRNA-seq. We aim to fill this critical knowledge gap by providing a comparative evaluation of a variety of commonly used dimensionality reduction methods for scRNA-seq studies. Specifically, we compare 18 different dimensionality reduction methods on 30 publicly available scRNA-seq datasets that cover a range of sequencing techniques and sample sizes. We evaluate the performance of different dimensionality reduction methods for neighborhood preserving in terms of their ability to recover features of the original expression matrix, and for cell clustering and lineage reconstruction in terms of their accuracy and robustness. We also evaluate the computational scalability of different dimensionality reduction methods by recording their computational cost. Based on the comprehensive evaluation results, we provide important guidelines for choosing dimensionality reduction methods for scRNA-seq data analysis.</p>
                <p class="card-text"><em>Shiquan Sun, Jiaqiang Zhu, Ying Ma, Xiang Zhou</em></p>
                <p class="card-text text-danger"><strong></strong></p>
                <p class="card-text"></p>
              </div>
              <div class="card-footer px-0 mx-auto text-center w-100">
                <p class="card-link text-nowrap"><strong><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1898-6">Genome Biology 2019</a></strong></p>
              </div>
            </div>
          </div>
        
      
    </div>

    <p>&nbsp;</p>

    <h2>List of publications</h2>
    
      <p>
        ExPRSweb: An online repository with polygenic risk scores for common health-related exposures<br>
        <em>Ying Ma, Snehal Patil, Xiang Zhou, Bhramar Mukherjee, Lars G Fritsche</em><br>
        <a href="https://www.cell.com/ajhg/pdf/S0002-9297(22)00404-9.pdf">The American Journal of Human Genetics 2022</a>
      </p>
    
      <p>
        Modeling zero inflation is not necessary for spatial transcriptomics<br>
        <em>Peiyao Zhao, Jiaqiang Zhu, Ying Ma, Xiang Zhou</em><br>
        <a href="https://doi.org/10.1186/s13059-022-02684-0">Genome Biology 2022</a>
      </p>
    
      <p>
        Spatially informed cell type deconvolution for spatial transcriptomics<br>
        <em>Ying Ma and Xiang Zhou</em><br>
        <a href="https://www.nature.com/articles/s41587-022-01273-7">Nature Biotechnology 2022</a>
      </p>
    
      <p>
        Genetic prediction of complex traits with polygenic scores: a statistical review<br>
        <em>Ying Ma and Xiang Zhou</em><br>
        <a href="https://www.cell.com/trends/genetics/fulltext/S0168-9525(21)00145-1">Trends in Genetics 2021</a>
      </p>
    
      <p>
        On cross-ancestry cancer polygenic risk scores<br>
        <em>Lars G Fritsche, Ying Ma, Daiwei Zhang, Maxwell Salvatore, Seunggeun Lee, Xiang Zhou, Bhramar Mukherjee</em><br>
        <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009670">PLoS genetics 2021</a>
      </p>
    
      <p>
        Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks<br>
        <em>Lars G Fritsche, Snehal Patil, Lauren J Beesley, Peter VandeHaar, Maxwell Salvatore, Ying Ma, Robert B Peng, Daniel Taliun, Xiang Zhou, Bhramar Mukherjee</em><br>
        <a href="https://www.cell.com/ajhg/pdf/S0002-9297(20)30320-7.pdf">The American Journal of Human Genetics 2021</a>
      </p>
    
      <p>
        Integrative differential expression and gene set enrichment analysis using summary statistics for scRNA-seq studies<br>
        <em>Ying Ma, Shiquan Sun, Xuequn Shang, Evan T Keller, Mengjie Chen, Xiang Zhou</em><br>
        <a href="https://www.nature.com/articles/s41467-020-15298-6">Nature communications 2020</a>
      </p>
    
      <p>
        Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis<br>
        <em>Shiquan Sun, Jiaqiang Zhu, Ying Ma, Xiang Zhou</em><br>
        <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1898-6">Genome Biology 2019</a>
      </p>
    
  </div>
</div>


    <div class="page__footer">
      <footer>
        <!-- start custom footer snippets -->
<a href="/sitemap/">Sitemap</a>
<!-- end custom footer snippets -->

        

<div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    
    
    
    
      <li><a href="http://github.com/https://github.com/YingMa0107"><i class="fab fa-github" aria-hidden="true"></i> GitHub</a></li>
    
    
    <li><a href="https://YingMa0107.github.io/feed.xml"><i class="fa fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2023 Ying Ma. Powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://github.com/academicpages/academicpages.github.io">AcademicPages</a>, a fork of <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    <script src="https://YingMa0107.github.io/assets/js/main.min.js"></script>




  <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');
</script>






  </body>
</html>

