

<!doctype html>
<html lang="en" class="no-js">
  <head>
    

<meta charset="utf-8">
<!-- Favicon Links Begin -->
<link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicon/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
<link rel="manifest" href="/assets/images/favicon/site.webmanifest">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" content="#ffffff">
<!-- Favicon Links End -->



<!-- begin SEO -->









<title>Publications - Ying Ma’s Personal Website</title>







<meta property="og:locale" content="en-US">
<meta property="og:site_name" content="Ying Ma's Personal Website">
<meta property="og:title" content="Publications">


  <link rel="canonical" href="https://YingMa0107.github.io/publications/">
  <meta property="og:url" content="https://YingMa0107.github.io/publications/">







  

  












  <script type="application/ld+json">
    {
      "@context" : "http://schema.org",
      "@type" : "Person",
      "name" : "Ying Ma",
      "url" : "https://YingMa0107.github.io",
      "sameAs" : null
    }
  </script>






<!-- end SEO -->


<link href="https://YingMa0107.github.io/feed.xml" type="application/atom+xml" rel="alternate" title="Ying Ma's Personal Website Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="https://YingMa0107.github.io/assets/css/main.css">

<meta http-equiv="cleartype" content="on">
    

<!-- start custom head snippets -->

<link rel="apple-touch-icon" sizes="57x57" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="60x60" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="72x72" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="76x76" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="114x114" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="120x120" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="144x144" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="152x152" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="180x180" href="https://YingMa0107.github.io/assets/images/favicon/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
<link rel="manifest" href="/assets/images/favicon/site.webmanifest">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" content="#ffffff">
<link rel="stylesheet" href="https://YingMa0107.github.io/assets/css/academicons.css"/>
<script type="text/x-mathjax-config"> MathJax.Hub.Config({ TeX: { equationNumbers: { autoNumber: "all" } } }); </script>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/latest.js?config=TeX-MML-AM_CHTML' async></script>

<!-- end custom head snippets -->

    <!-- Favicon Links Begin -->
    <link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicon/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
    <link rel="manifest" href="/assets/images/favicon/site.webmanifest">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="theme-color" content="#ffffff">
  </head>

  <body>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->
    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <button><div class="navicon"></div></button>
        <ul class="visible-links">
          <li class="masthead__menu-item masthead__menu-item--lg"><a href="https://YingMa0107.github.io/">Ying Ma's Personal Website</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://YingMa0107.github.io/publications/">Publications</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://YingMa0107.github.io/talks/">Talks</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://YingMa0107.github.io/research/">Research</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://YingMa0107.github.io/software/">Software</a></li>
          
        </ul>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>

    <style>
  .card {
    box-shadow: 2px 2px 5px #888888 inset
  }
</style>
<div id="gridid" class="col-sm-12">
  <h1 id="publications">Publications</h1>

<div class="row row-cols-1 row-cols-xl-2">

















<div class="col mb-4">

 <div class="card h-100 d-flex flex-column justify-content-between bg-light">
  <div class="card-body clearfix">
  <pubtit class="card-title">Variational mixtures of ODEs for inferring cellular gene expression dynamics</pubtit>
  <img src="https://YingMa0107.github.io/images/pubpic/ICML.png" class="img-fluid float-left w-33 d-inline-block" />
  <p class="card-text">By using a simple family of ODEs informed by the biochemistry of gene expression to constrain the likelihood of a deep generative model, we can simultaneously infer the latent time and latent state of each cell and predict its future gene expression state. The model can be interpreted as a mixture of ODEs whose parameters vary continuously across a latent space of cell states.</p>
  <p class="card-text"><em>Gu Y, Blaauw D, Welch JD</em></p>
  <p class="card-text text-danger"><strong> </strong></p>
  <p class="card-text"> </p>
  </div>
  <div class="card-footer px-0 mx-auto text-center w-100">
  <p class="card-link text-nowrap"><strong><a href="https://icml.cc/virtual/2022/spotlight/16630">ICML 2022</a></strong></p>
  </div>
 </div>
</div>





<div class="col mb-4">

 <div class="card h-100 d-flex flex-column justify-content-between bg-light">
  <div class="card-body clearfix">
  <pubtit class="card-title">UINMF performs mosaic integration of single-cell multi-omic datasets using nonnegative matrix factorization</pubtit>
  <img src="https://YingMa0107.github.io/images/pubpic/uinmf.png" class="img-fluid float-left w-33 d-inline-block" />
  <p class="card-text">Integration analyses often involve datasets with partially overlapping features, including both shared features that occur in all datasets and features exclusive to a single experiment. To address this limitation, we derive a novel nonnegative matrix factorization algorithm for integrating single-cell datasets containing both shared and unshared features.</p>
  <p class="card-text"><em>Kriebel AR, Welch JD</em></p>
  <p class="card-text text-danger"><strong> </strong></p>
  <p class="card-text"> </p>
  </div>
  <div class="card-footer px-0 mx-auto text-center w-100">
  <p class="card-link text-nowrap"><strong><a href="https://rdcu.be/cGFpq">Nature Communications 13:780 (2022)</a></strong></p>
  </div>
 </div>
</div>







<div class="col mb-4">

 <div class="card h-100 d-flex flex-column justify-content-between bg-light">
  <div class="card-body clearfix">
  <pubtit class="card-title">Multi-omic single-cell velocity models epigenome–transcriptome interactions and improves cell fate prediction</pubtit>
  <img src="https://YingMa0107.github.io/images/pubpic/multivelo.png" class="img-fluid float-left w-33 d-inline-block" />
  <p class="card-text">Cells differentiate to their final fates through sequential epigenetic and transcriptional changes. A mathematical model fit on multi-omic single-cell data yields insights into the temporal relationships between chromatin accessibility and gene expression during cell differentiation.</p>
  <p class="card-text"><em>Li C, Virgilio MC, Collins KL, Welch JD</em></p>
  <p class="card-text text-danger"><strong> </strong></p>
  <p class="card-text"> Also accepted at RECOMB 2022.</p>
  </div>
  <div class="card-footer px-0 mx-auto text-center w-100">
  <p class="card-link text-nowrap"><strong><a href="https://rdcu.be/cXuUu">Nature Biotechnology 2022.</a></strong></p>
  </div>
 </div>
</div>









<div class="col mb-4">

 <div class="card h-100 d-flex flex-column justify-content-between bg-light">
  <div class="card-body clearfix">
  <pubtit class="card-title">A transcriptomic and epigenomic cell atlas of the mouse primary motor cortex</pubtit>
  <img src="https://YingMa0107.github.io/images/pubpic/atlas.png" class="img-fluid float-left w-33 d-inline-block" />
  <p class="card-text">The authors use SingleCellFusion and LIGER to assemble scRNA-seq, snRNA-seq, snmC-seq, and snATAC-seq into an integrated atlas of the diverse cell types in the mouse primary motor cortex.</p>
  <p class="card-text"><em>Zizhen Yao, Hanqing Liu, Fangming Xie, Stephan Fischer, ..., Bosiljka Tasic, Joshua D. Welch, Joseph R. Ecker, Evan Z Macosko, Bing Ren, BRAIN Initiative Cell Census Network (BICCN), Hongkui Zeng, Eran A. Mukamel.</em></p>
  <p class="card-text text-danger"><strong> </strong></p>
  <p class="card-text"> </p>
  </div>
  <div class="card-footer px-0 mx-auto text-center w-100">
  <p class="card-link text-nowrap"><strong><a href="https://www.nature.com/articles/s41586-021-03500-8">Nature 598, 103–110 (2021)</a></strong></p>
  </div>
 </div>
</div>

















<div class="col mb-4">

 <div class="card h-100 d-flex flex-column justify-content-between bg-light">
  <div class="card-body clearfix">
  <pubtit class="card-title">MichiGAN: Sampling from disentangled representations of single-cell data using generative adversarial networks</pubtit>
  <img src="https://YingMa0107.github.io/images/pubpic/michigan.png" class="img-fluid float-left w-33 d-inline-block" />
  <p class="card-text">We develop MichiGAN, a novel neural network that combines the strengths of VAEs and GANs to sample from disentangled representations without sacrificing data generation quality. MichiGAN allows us to manipulate semantically distinct aspects of cellular identity and predict single-cell gene expression response to drug treatment.</p>
  <p class="card-text"><em>Yu H, Welch JD</em></p>
  <p class="card-text text-danger"><strong> </strong></p>
  <p class="card-text"> Also presented at Learning Meaningful Representations of Life (LMRL 2020), Workshop at NeurIPS 2020.</p>
  </div>
  <div class="card-footer px-0 mx-auto text-center w-100">
  <p class="card-link text-nowrap"><strong><a href="https://doi.org/10.1186/s13059-021-02373-4">Genome Biology 22, 158 (2021)</a></strong></p>
  </div>
 </div>
</div>





<div class="col mb-4">

 <div class="card h-100 d-flex flex-column justify-content-between bg-light">
  <div class="card-body clearfix">
  <pubtit class="card-title">Iterative Single-Cell Multi-Omic Integration Using Online Learning</pubtit>
  <img src="https://YingMa0107.github.io/images/pubpic/online-inmf.png" class="img-fluid float-left w-33 d-inline-block" />
  <p class="card-text">Here we describe online integrative non-negative matrix factorization (iNMF), an algorithm for integrating large, diverse and continually arriving single-cell datasets. Our approach scales to arbitrarily large numbers of cells using fixed memory, iteratively incorporates new datasets as they are generated and allows many users to simultaneously analyze a single copy of a large dataset by streaming it over the internet. Iterative data addition can also be used to map new data to a reference dataset.</p>
  <p class="card-text"><em>Gao C, Liu J, Kriebel AR, Preissl S, Luo C, Castanon R, Sandoval J, Rivkin A, Nery JR, Behrens MM, Ecker JR, Ren B, Welch JD</em></p>
  <p class="card-text text-danger"><strong> </strong></p>
  <p class="card-text"> </p>
  </div>
  <div class="card-footer px-0 mx-auto text-center w-100">
  <p class="card-link text-nowrap"><strong><a href="https://doi.org/10.1038/s41587-021-00867-x">Nature Biotechnology 39, 1000–1007 (2021)</a></strong></p>
  </div>
 </div>
</div>











<div class="col mb-4">

 <div class="card h-100 d-flex flex-column justify-content-between bg-light">
  <div class="card-body clearfix">
  <pubtit class="card-title">Single-cell multi-omic integration compares and contrasts features of brain cell identity</pubtit>
  <img src="https://YingMa0107.github.io/images/pubpic/liger-cell.jpg" class="img-fluid float-left w-33 d-inline-block" />
  <p class="card-text">To flexibly model single-cell datasets, we developed LIGER, an algorithm that delineates shared and dataset-specific features of cell identity. We applied it to four diverse and challenging analyses of human and mouse brain cells. Integrative analyses using LIGER promise to accelerate investigations of cell-type definition, gene regulation, and disease states.</p>
  <p class="card-text"><em>Welch JD\*, Kozareva V, Ferreira A, Vanderburg C, Martin C, Macosko EZ\*</em></p>
  <p class="card-text text-danger"><strong> </strong></p>
  <p class="card-text"> \*Co-Senior Authors</p>
  </div>
  <div class="card-footer px-0 mx-auto text-center w-100">
  <p class="card-link text-nowrap"><strong><a href="https://www.sciencedirect.com/science/article/pii/S0092867419305045">Cell. Volume 177, Issue 7, 13 June 2019, Pages 1873-1887.e17</a></strong></p>
  </div>
 </div>
</div>







<div class="col mb-4">

 <div class="card h-100 d-flex flex-column justify-content-between bg-light">
  <div class="card-body clearfix">
  <pubtit class="card-title">Single-cell transcriptomics reconstructs fate conversion from fibroblast to cardiomyocyte</pubtit>
  <img src="https://YingMa0107.github.io/images/pubpic/icm.png" class="img-fluid float-left w-33 d-inline-block" />
  <p class="card-text">We used single-cell RNA sequencing to analyse global transcriptome changes at early stages during the reprogramming of mouse fibroblasts into induced cardiomyocytes (iCMs). Using unsupervised dimensionality reduction and clustering algorithms, we identified molecularly distinct subpopulations of cells during reprogramming. We also constructed routes of iCM formation, and delineated the relationship between cell proliferation and iCM induction.</p>
  <p class="card-text"><em>Liu Z\*, Wang L\*, Welch JD\*, Ma H, Zhou Y, Vaseghi HR, Yu S, Wall JB, Alimohamadi S, Zheng M, Yin C, Shen W, Prins JF, Liu J, Qian L</em></p>
  <p class="card-text text-danger"><strong> </strong></p>
  <p class="card-text"> \*Equal contribution</p>
  </div>
  <div class="card-footer px-0 mx-auto text-center w-100">
  <p class="card-link text-nowrap"><strong><a href="https://www.nature.com/articles/nature24454">Nature (02 November 2017). 551, 100–104 doi:10.1038/nature24454</a></strong></p>
  </div>
 </div>
</div>



























</div>

<p> &nbsp; </p>

<h2 id="full-list-of-publications">Full List of publications</h2>

<p>Multiplying insights from perturbation experiments: predicting new perturbation combinations <br />
  <em>Welch JD </em><br /><a href="https://www.embopress.org/doi/full/10.15252/msb.202311667">Molecular Systems Biology 2023</a></p>

<p>Bone marrow endosteal stem cells dictate active osteogenesis and aggressive tumorigenesis <br />
  <em>Matsushita Y*, Liu J*, Ka Yan Chu A, Tsutsumi-Arai C, Nagata M, Arai Y, Ono W, Yamamoto K, Saunders TL, Welch JD*, Ono N* </em><br /><a href="https://rdcu.be/daHuT">Nature Communications 2023</a></p>

<p>HIV-1 Vpr combats the PU.1-driven antiviral response in primary human macrophages <br />
  <em>Virgilio MC,  Disbennett WM,  Chen T, Lubow Jay, Welch JD, Collins KL </em><br /><a href="https://www.biorxiv.org/content/10.1101/2023.03.21.533528v1">bioRxiv 2023</a></p>

<p>MorphNet predicts cell morphology from single-cell gene expression <br />
  <em>Lee H, Welch JD </em><br /><a href="https://www.biorxiv.org/content/10.1101/2022.10.21.513201v1">bioRxiv 2022</a></p>

<p>A bilevel optimization method for tensor recovery under metric learning constraints <br />
  <em>Bagherian M, Tarzanagh DA, Dinov I, Welch JD </em><br /><a href="https://arxiv.org/abs/2209.00545">arXiv 2022</a></p>

<p>PerturbNet predicts single-cell responses to unseen chemical and genetic perturbations <br />
  <em>Yu H, Welch JD </em><br /><a href="https://www.biorxiv.org/content/10.1101/2022.07.20.500854v1">bioRxiv 2022</a></p>

<p>Bayesian inference of RNA velocity from multi-lineage single-cell data <br />
  <em>Gu Y, Blaauw D, Welch JD </em><br /><a href="https://www.biorxiv.org/content/10.1101/2022.07.08.499381v1">bioRxiv 2022</a></p>

<p>Variational mixtures of ODEs for inferring cellular gene expression dynamics <br />
  <em>Gu Y, Blaauw D, Welch JD </em><br /><a href="https://icml.cc/virtual/2022/spotlight/16630">ICML 2022</a></p>

<p>UINMF performs mosaic integration of single-cell multi-omic datasets using nonnegative matrix factorization <br />
  <em>Kriebel AR, Welch JD </em><br /><a href="https://rdcu.be/cGFpq">Nature Communications 13:780 (2022)</a></p>

<p>PyLiger: Scalable single-cell multi-omic data integration in Python <br />
  <em>Lu L, Welch JD </em><br /><a href="https://doi.org/10.1093/bioinformatics/btac190">Bioinformatics, btac190, March 2022</a></p>

<p>Multi-omic single-cell velocity models epigenome–transcriptome interactions and improves cell fate prediction <br />
  <em>Li C, Virgilio MC, Collins KL, Welch JD </em><br /><a href="https://rdcu.be/cXuUu">Nature Biotechnology 2022.</a></p>

<p>SquiggleNet: Real-Time, Direct Classification of Nanopore Signals <br />
  <em>Bao Y, Welch JD </em><br /><a href="https://rdcu.be/cAj1A">Genome Biology 22:298 (2021)</a></p>

<p>A multimodal cell census and atlas of the mammalian primary motor cortex <br />
  <em>BRAIN Initiative Cell Census Network (BICCN) </em><br /><a href="https://www.nature.com/articles/s41586-021-03950-0">Nature 598, 86–102 (2021)</a></p>

<p>A transcriptomic and epigenomic cell atlas of the mouse primary motor cortex <br />
  <em>Zizhen Yao, Hanqing Liu, Fangming Xie, Stephan Fischer, …, Bosiljka Tasic, Joshua D. Welch, Joseph R. Ecker, Evan Z Macosko, Bing Ren, BRAIN Initiative Cell Census Network (BICCN), Hongkui Zeng, Eran A. Mukamel. </em><br /><a href="https://www.nature.com/articles/s41586-021-03500-8">Nature 598, 103–110 (2021)</a></p>

<p>Functional coordination of non-myocytes plays a key role in adult zebrafish heart regeneration <br />
  <em>Ma H, Liu Z, Yang Y, Feng D, Dong Y, Garbutt T, Hu Z, Wang L, Luan C, Cooper C, Li Y, Welch JD*, Qian L*, Liu J* </em><br /><a href="https://www.embopress.org/doi/abs/10.15252/embr.202152901">EMBO Rep (2021) e52901</a></p>

<p>G-CSF secreted by mutant IDH1 glioma stem cells abolishes myeloid cell immunosuppression and enhances the efficacy of immunotherapy <br />
  <em>Alghamri MS, Avvari RP, Thalla R, Kamran N, Zhang L, Ventosa M, Taher A, Faisal SM, Núñez FJ, Fabiani MBG, Haase S, Carney S, Orringer D, Hervey-Jumper S, Heth J, Patil PG, Al-Holou WN, Eddy K, Merajver S, Ulintz PJ, Welch JD, Gao C, Liu J, Núñez G, Hambardzumyan D, Lowenstein PR, Castro M </em><br /><a href="https://www.science.org/doi/10.1126/sciadv.abh3243">Science Advances Vol 7, Issue 40 (2021)</a></p>

<p>Transcription factor FOXF1 identifies compartmentally distinct mesenchymal cells with a role in lung allograft fibrogenesis <br />
  <em>Braeuer RR, Walker NM, Misumi K, Mazzoni-Putman S, Aoki Y, Liao R, Vittal R, Kleer GG, Wheeler DS, Sexton JZ, Farver CF, Welch JD, Lama VN </em><br /><a href="https://www.jci.org/articles/view/147343">Journal of Clinical Investigation. doi:10.1172/JCI147343</a></p>

<p>Community-wide hackathons to identify central themes in single-cell multi-omics <br />
  <em>Lê Cao KA, Abadi AJ, Davis-Marcisak EF, Hsu L, Arora A, Coullomb A, Deshpande A, Feng Y, Jeganathan P, Loth M, Meng C, Mu W, Pancaldi V, Sankaran K, Singh A, Sodicoff JS, Stein-O’Brien GL, Subramanian A, Welch JD, You Y, Argelaguet R, Carey VJ, Dries R, Greene CS, Holmes S, Love MI, Ritchie ME, Yuan GC, Culhane AC, Fertig E </em><br /><a href="https://doi.org/10.1186/s13059-021-02433-9">Genome Biology 22, 220 (2021)</a></p>

<p>Single-cell transcriptomic analysis reveals developmental relationships and specific markers of mouse periodontium cellular subsets. Frontiers in Dental Medicine <br />
  <em>Nagata M, Ka Yan Chu A, Ono N, Welch JD, Ono W </em><br /><a href="https://www.frontiersin.org/articles/10.3389/fdmed.2021.679937/full">Frontiers in Dental Medicine 2:56 (2021)</a></p>

<p>Intercellular interactions of an adipogenic CXCL12‐expressing stromal cell subset in murine bone marrow <br />
  <em>Matsushita Y, Ka Yan Chu A, Ono W, Welch JD, Ono N </em><br /><a href="https://doi.org/10.1002/jbmr.4282">Journal of Bone and Mineral Research 2021</a></p>

<p>MichiGAN: Sampling from disentangled representations of single-cell data using generative adversarial networks <br />
  <em>Yu H, Welch JD </em><br /><a href="https://doi.org/10.1186/s13059-021-02373-4">Genome Biology 22, 158 (2021)</a></p>

<p>Iterative Single-Cell Multi-Omic Integration Using Online Learning <br />
  <em>Gao C, Liu J, Kriebel AR, Preissl S, Luo C, Castanon R, Sandoval J, Rivkin A, Nery JR, Behrens MM, Ecker JR, Ren B, Welch JD </em><br /><a href="https://doi.org/10.1038/s41587-021-00867-x">Nature Biotechnology 39, 1000–1007 (2021)</a></p>

<p>Jointly defining cell types from multiple single-cell datasets using LIGER <br />
  <em>Liu J*, Gao C*, Sodicoff J, Kozareva V, Macosko EZ, Welch JD </em><br /><a href="https://doi.org/10.1038/s41596-020-0391-8">Nature Protocols (2020)</a></p>

<p>A Wnt-mediated transformation of the bone marrow stromal cell identity orchestrates skeletal regeneration <br />
  <em>Matsushita Y, Nagata M, Kozloff K, Welch J, Mizuhashi K, Tokavanich N, Hallett S, Link D, Nagasawa T, Ono W, and Ono N </em><br /><a href="https://www.nature.com/articles/s41467-019-14029-w">Nature Communications. 11, 332 (2020)</a></p>

<p>Single-Cell Transcriptomic Analyses of Cell Fate Transitions during Human Cardiac Reprogramming <br />
  <em>Zhou Y, Liu Z, Welch JD, Gao X, Wang L, Garbutt T, Keepers B, Ma H, Prins JF, Shen W, Liu J, Qian L </em><br /><a href="https://www.sciencedirect.com/science/article/pii/S1934590919302243">Stem Cell. 2019 Jun 12</a></p>

<p>Single-cell multi-omic integration compares and contrasts features of brain cell identity <br />
  <em>Welch JD*, Kozareva V, Ferreira A, Vanderburg C, Martin C, Macosko EZ* </em><br /><a href="https://www.sciencedirect.com/science/article/pii/S0092867419305045">Cell. Volume 177, Issue 7, 13 June 2019, Pages 1873-1887.e17</a></p>

<p>Slide seq: A Scalable Technology for Measuring Genome-Wide Expression at High Spatial Resolution <br />
  <em>Rodriques SG<em>, Stickels RR</em>, Goeva A Martin CA, Murray E, Vanderburg CR, Welch JD, Chen LM, Chen F+, Macosko EZ+ </em><br /><a href="http://science.sciencemag.org/content/sci/363/6434/1463.full.pdf">Science 29 Mar 2019: Vol. 363, Issue 6434, pp. 1463-1467</a></p>

<p>Single-cell transcriptomics reconstructs fate conversion from fibroblast to cardiomyocyte <br />
  <em>Liu Z*, Wang L*, Welch JD*, Ma H, Zhou Y, Vaseghi HR, Yu S, Wall JB, Alimohamadi S, Zheng M, Yin C, Shen W, Prins JF, Liu J, Qian L </em><br /><a href="https://www.nature.com/articles/nature24454">Nature (02 November 2017). 551, 100–104 doi:10.1038/nature24454</a></p>

<p>MATCHER: manifold alignment reveals correspondence between single cell transcriptome and epigenome dynamics <br />
  <em>Welch JD, Hartemink AJ, Prins JF </em><br /><a href="https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-017-1269-0">Genome Biology 2017.</a></p>

<p>Selective Single Cell Isolation for Genomics Using Microraft Arrays <br />
  <em>Welch JD<em>, Williams LA</em>, DiSalvo M*, Brandt AT, Marayati R, Sims CE, Allbritton NL, Prins JF, Yeh JJ, Jones CD </em><br /><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5041489/pdf/gkw700.pdf">Nucleic Acids Research 2016</a></p>

<p>A subset of replication-dependent histone mRNAs are expressed as polyadenylated RNAs in terminally differentiated tissues <br />
  <em>Lyons SM, Cunningham CH, Welch JD, Groh B, Guo AY, Wei B, Whitfield ML, Xiong Y, Marzluff WF </em><br /><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5100578/pdf/gkw620.pdf">Nucleic Acids Research 2016</a></p>

<p>SLICER: Inferring branched, nonlinear cellular trajectories from single cell RNA-seq data <br />
  <em>Welch JD, Hartemink AJ, Prins JF </em><br /><a href="https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-016-0975-3">Genome Biology 2016, 17</a></p>

<p>Robust Detection of Alternative Splicing in a Population of Single Cells <br />
  <em>Welch JD, Hu Y, Prins JF </em><br /><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4856971/pdf/gkv1525.pdf">Nucleic Acids Research 2016</a></p>

<p>A multi-protein occupancy map of the mRNP on the 3 end of histone mRNAs <br />
  <em>Brooks L, Lyons SM, Mahoney JM, Welch JD, Liu Z, Marzluff WM, Whitfield ML </em><br /><a href="https://rnajournal.cshlp.org/content/21/11/1943.full.pdf+html">RNA 2015</a></p>

<p>EnD-Seq and AppEnD: Sequencing 3 ends to identify nontemplated tails and degradation intermediates <br />
  <em>Welch JD<em>, Slevin MK</em>, Tatomer D, Duronio RJ, Prins JF, Marzluff WF </em><br /><a href="https://rnajournal.cshlp.org/content/21/7/1375.full.pdf+html">RNA 2015</a></p>

<p>Pseudogenes Transcribed in Breast Invasive Carcinoma Show Subtype-Specific Expression and ceRNA Potential <br />
  <em>Welch JD, Baran-Gale J, Perou C, Sethupathy P, Prins JF </em><br /><a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-015-1227-8">BMC Genomics 2015</a></p>

<p>Deep Sequencing Shows Multiple Oligouridylations Are Required for 3′ to 5′ Degradation of Histone mRNAs on Polyribosomes <br />
  <em>Slevin MK, Meaux S, Welch JD, Bigler R, Miliani de Marval PL, Su W, Rhoads RE, Prins JF, Marzluff WF </em><br /><a href="https://www.cell.com/action/showPdf?pii=S1097-2765%2814%2900199-3">Molecular cell 53 (6), 1020-1030. March 2014</a></p>

<p>WordSeeker: concurrent bioinformatics software for discovering genome-wide patterns and word-based genomic signatures <br />
  <em>Lichtenberg J, Kurz K, Liang X, Al-ouran R, Neiman L, Nau LJ, Welch JD, Jacox E, Bitterman T, Ecker K, Elnitski L, Drews F, Lee SS, Welch LR </em><br /><a href="">BMC Bioinformatics. 2010 Dec 21;11 Suppl 12</a></p>

<p>The word landscape of the non-coding segments of the Arabidopsis thaliana genome. BMC Genomics <br />
  <em>Lichtenberg J, Yilmaz A, Welch JD, Kurz K, Liang X, Drews F, Ecker K, Lee SS, Geisler M, Grotewold E, Welch LR </em><br /><a href="">BMC Genomics. 2009 Oct 8</a></p>

<p>Word-based characterization of promoters involved in human DNA repair pathways <br />
  <em>Lichtenberg J, Jacox E, Welch JD, Kurz K, Liang X, Yang MQ, Drews F, Ecker K, Lee SS, Elnitski L, Welch LR </em><br /><a href="">BMC Genomics. 2009 Jul 7;10 Suppl 1</a></p>


</div>


    <div class="page__footer">
      <footer>
        <!-- start custom footer snippets -->
<a href="/sitemap/">Sitemap</a>
<!-- end custom footer snippets -->

        

<div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    
    
    
    
      <li><a href="http://github.com/https://github.com/YingMa0107"><i class="fab fa-github" aria-hidden="true"></i> GitHub</a></li>
    
    
    <li><a href="https://YingMa0107.github.io/feed.xml"><i class="fa fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2023 Ying Ma. Powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://github.com/academicpages/academicpages.github.io">AcademicPages</a>, a fork of <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    <script src="https://YingMa0107.github.io/assets/js/main.min.js"></script>




  <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');
</script>






  </body>
</html>

